client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -56,17 +56,36 @@ const DesignSelector: FunctionComponent = () => {
 	const [ isDialogVisible, setIsDialogVisible ] = useState( hasSelectedDesign );
 	const [ cp, setCp ] = useState< number >();
 
+	const headingContainer = useRef< HTMLDivElement >( null );
+	const selectionTransitionShift = useRef< number >( 0 );
+	useLayoutEffect( () => {
+		if ( headingContainer.current ) {
+			// We'll use this height to move the heading up out of the viewport.
+			// Gutenberg editor adds 50px
+			const rect = headingContainer.current.getBoundingClientRect();
+			selectionTransitionShift.current = rect.height + rect.y - 50;
+		}
+	}, [ selectedDesign ] );
+
 	const dialogId = 'page-selector-modal';
 
 	const descriptionOnRight: boolean =
 		!! selectedDesign &&
 		designs.findIndex( ( { slug } ) => slug === selectedDesign.slug ) % 2 === 0;
 
 	return (
-		<div className={ classnames( 'design-selector', { 'has-selected-design': selectedDesign } ) }>
+		<div
+			className="design-selector"
+			style={
+				selectedDesign && {
+					transform: `translate3d( 0,  -${ selectionTransitionShift.current }px, 0 )`,
+				}
+			}
+		>
 			<div
-				className="design-selector__header-container"
+				onClick={ () => resetState() }
```

Comment:
Do we need binding here?

```suggestion
				onClick={ resetState }
```

Reply:
Huzzah for hooks, we don't need the binding anymore :) But I think that `onClick` is only there because of a sloppy rebase on my part. It's been removed on `master`